apps/coscrad-frontend/src/app/routes/get-resource-type-label-for-routes.ts

```diff
@@ -0,0 +1,23 @@
+import { ResourceType } from '@coscrad/api-interfaces';
+
+const lookupTable: { [K in ResourceType]: string } = {
+    [ResourceType.bibliographicReference]: 'BibliographicReferences',
+    [ResourceType.book]: 'Books',
+    [ResourceType.mediaItem]: 'MediaItems',
+    [ResourceType.photograph]: 'Photographs',
+    [ResourceType.song]: 'Songs',
+    [ResourceType.spatialFeature]: 'Map',
+    [ResourceType.term]: 'Terms',
+    [ResourceType.transcribedAudio]: 'Transcripts',
+    [ResourceType.vocabularyList]: 'VocabularyLists',
```

Comment:
It might be nice to consider having a default plural (add an s) behaviour and registering exceptions only. We can think about that later.